### DISCUSSION TITLE
I would like to be able to input the video from Orange 5Plus via HDMI IN to OBS Studio.

### DISCUSSION THREAD

**Discussion by ironiris** [Joshua-Riek/ubuntu-rockchip#139] [Ideas]
I would like to be able to input the video from Orange 5Plus via HDMI IN to OBS Studio.
Usually, HDMI capture boards that support 4K@60fps are quite expensive and require a laptop or PC for streaming, but the hardware environment of Orange 5Plus has a built-in hardware-based video encoder, so the combination of a capture board and a laptop can be handled with just Orange 5Plus.

**Comment by Joshua-Riek** [comment]
Orange Pi contacted me about the opi5 plus, I should have it in my hands sometime next week or the week after. I only hope to get it early because Rockchip released a new 5.10.160 kernel which has some interesting changes and I doubt Orange Pi will update its kernel branch, so I will add the device to the latest Rockchip kernel myself.

**Reply by ironiris** [reply]
I'm looking forward to it. :)
